docs/application_deployment.md

```diff
@@ -78,8 +78,10 @@ The logs of a single AEA or node can then be inspected with `docker logs {contai
 ## Building & tagging local images
 
 ```bash
+export SERVICE_ID=author_name/service_name
 export VERSION=0.1.0
-make build-images
+autonomy deploy build image ${SERVICE_ID} --dependencies
+autonomy deploy build image ${SERVICE_ID} --version ${VERSION}
```

Comment:
Can we add in future  the help output of the command in order to ensure that we demonstrate the entire functionality.
i.e.
 --skip-images
-- kubernetes